git_machete/tests/functional/test_machete.py

```diff
@@ -37,39 +41,43 @@ def new_repo(self, *args: str) -> "SandboxSetup":
             self.execute("git init")
         return self
 
-    def new_branch(self, branch_name: str) -> "SandboxSetup":
+    def new_branch(self, branch_name: str) -> "GitRepositorySandbox":
         self.execute(f"git checkout -q -b {branch_name}")
         return self
 
-    def check_out(self, branch: str) -> "SandboxSetup":
+    def new_root_branch(self, branch_name: str) -> "GitRepositorySandbox":
+        self.execute(f"git checkout --orphan {branch_name}")
+        return self
+
+    def check_out(self, branch: str) -> "GitRepositorySandbox":
         self.execute(f"git checkout -q {branch}")
         return self
 
-    def commit(self, message: str) -> "SandboxSetup":
+    def commit(self, message: str = "SomeCommitMsg") -> "GitRepositorySandbox":
```

Comment:
Nit: to make it look more like an actual commit message:
```suggestion
    def commit(self, message: str = "Some commit message") -> "GitRepositorySandbox":
```

Reply:
Accepted